tensorboard/webapp/metrics/views/card_renderer/histogram_card_container.ts

```diff
@@ -122,9 +127,9 @@ export class HistogramCardContainer implements CardRenderer, OnInit {
   }
 
   onFullSizeToggle() {
-    this.showFullSize = !this.showFullSize;
-    this.fullWidthChanged.emit(this.showFullSize);
-    this.fullHeightChanged.emit(this.showFullSize);
+    this.store.dispatch(
+      metricsCardTableExpansionToggled({cardId: this.cardId})
```

Comment:
I think this needs to dispatch a different event, maybe `metricsCardFullSizeToggled`? That event should set the value of `CardState.showFullSize` and, if true, the value of `CardState.tableExpanded`

Reply:
Not quite understand the "if true" part but I did it this way:
When full size icon is toggled, we have fullWdith and  tableExpanded  both to be true. Then when the icon is toggled again, I think both should be collapsed. An extra click to collapse the table doesn't seem intuitive.
Open to discuss but this logic looks good to me for now.